db/migrate/20151111165020_rename_miq_search_db.rb

```diff
@@ -1,4 +1,8 @@
 class RenameMiqSearchDb < ActiveRecord::Migration
+  class MiqSearch < ActiveRecord::Base
```

Comment:
I wish Rails would just disable loading models via autoload in migrations.  </rant>